conjure-java-jersey-server/src/test/java/com/palantir/conjure/java/server/jersey/JsonExceptionMapperTest.java

```diff
@@ -18,21 +18,30 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.codahale.metrics.Meter;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 import com.palantir.conjure.java.api.errors.ErrorType;
 import com.palantir.conjure.java.serialization.ObjectMappers;
+import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
+import com.palantir.tritium.metrics.registry.MetricName;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
 import javax.ws.rs.core.Response;
 import org.junit.Test;
 
 public final class JsonExceptionMapperTest {
 
-    private final JsonExceptionMapper<RuntimeException> mapper = new JsonExceptionMapper<RuntimeException>() {
-        @Override
-        ErrorType getErrorType(RuntimeException _exception) {
-            return ErrorType.INVALID_ARGUMENT;
-        }
-    };
+    private final TaggedMetricRegistry taggedMetrics = new DefaultTaggedMetricRegistry();
+    private final Meter internalExceptionMeter = taggedMetrics.meter(MetricName.builder()
+            .safeName("conjure.server.exception")
+            .build());
```

Comment:
maybe just make an anonymous meter here instead e.g. `new Meter()` (as this name is now out of sync), then further down you can actually assert that it was incremented when you spect it to!

Reply:
Done, and added another test below